ctx.go

```diff
@@ -960,62 +1211,59 @@ func (ctx *Ctx) Subdomains(offset ...int) []string {
 }
 
 // Stale is not implemented yet, pull requests are welcome!
-func (ctx *Ctx) Stale() bool {
+func (ctx *Context) Stale() bool {
 	return !ctx.Fresh()
 }
 
 // Status sets the HTTP status for the response.
-// This method is chainable.
-func (ctx *Ctx) Status(status int) *Ctx {
-	ctx.Fasthttp.Response.SetStatusCode(status)
-	return ctx
+func (ctx *Context) Status(status int) {
+	ctx.Fasthttp().Response.SetStatusCode(status)
 }
 
 // Type sets the Content-Type HTTP header to the MIME type specified by the file extension.
-func (ctx *Ctx) Type(extension string, charset ...string) *Ctx {
+func (ctx *Context) Type(extension string, charset ...string) {
```

Comment:
why did you remove this return value here ? is this not implementable in the interface ?